src/Microsoft.DotNet.Build.Tasks/PackageFiles/CodeCoverage.targets

```diff
@@ -5,8 +5,8 @@
     Code coverage package versions go here and in the test-runtime-packages.config
   -->
   <PropertyGroup>
-    <OpenCoverVersion>4.6.476-rc</OpenCoverVersion>
-    <ReportGeneratorVersion>2.4.0</ReportGeneratorVersion>
+    <OpenCoverVersion>4.6.519</OpenCoverVersion>
```

Comment:
While is great to fully encapsulate everything into BuildTools I think it has some drawbacks when it comes to all these version numbers. I think BuildTools needs to have a default version defined that works with the tools and targets that we use from BuildTools itself but all the individual specific versions seem more like repo level configuration. So I think making them extensible in BuildTools and overriding them in the repo dir.props seems like the right thing to do for these cases.